File: pgm/ABC_Dart/ABC_397_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final double X = double.parse(stdin.readLineSync()!);
  final double dHigh = 38.0;
  final double dFever = 37.5;
  final int high = 1;
  final int fever = 2;
  final int normal = 3;
  if (dHigh <= X) {
    print(high);
  } else if (dFever <= X) {
    print(fever);
  } else {
    print(normal);
  }
  exit(0);
}

File: pgm/ABC_PHP/ABC_397_A.php
<?php
$stream = fopen('php://stdin', 'r');
$line = fgets($stream);
fclose($stream);
const dHigh = 38.0;
const dFever = 37.5;
const high = 1;
const fever = 2;
const normal = 3;
$X = (float) trim($line);
echo (dHigh <= $X ? high : (dFever <= $X ? fever : normal)) . PHP_EOL;
exit(0);
